initial implementaion of installing backend (#16)
DroneML/DroneML: README.md

In QGIS, go to `Plugins` > `Python Console`.

Then, go to `Plugins` > `Manage and Install Plugins...` > `Install from ZIP` and select the downloaded `droneml.zip` file   

### Manual Installation if install from ZIP fails

Clone the repository of DroneML from GitHub:

```bash

> git clone git@github.com:DroneML/segmentmytif.git

Reviewer: ```suggestion
git clone git@github.com:DroneML/droneml.git
```

```

> **Before ** copy the clonne `droneml` folder to the QGIS plugins directory. Examples on different operating systems:

Reviewer: ```suggestion
**Before ** copying the cloned `droneml` folder to the QGIS plugins directory. Examples on different operating systems:
```

> - Windows: `C:\Users\USER\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\`

Reviewer: ```suggestion
- Windows: `C:\Users\<USER>\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\`
```

- Linux: `~/.local/share/QGIS/QGIS3/profiles/default/python/plugins/`
- MacOS: `~/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/`

Then, go to `Plugins` > `Manage and Install Plugins...`. You will see the DroneML plugin in the list of installed plugins.
By checking the box, QGIS will start to install the Python dependencies required by the plugin.

> A command prompt will appear, showing the progress of the installation. Upon completion, the plugin will be ready to use.

Reviewer: nice description for nervous users :-)
